extensions/amp-story/1.0/amp-story-grid-layer.js

```diff
@@ -168,4 +170,15 @@ export class AmpStoryGridLayer extends AmpStoryBaseLayer {
     }
     setStyles(element, assertDoesNotContainDisplay(styles));
   }
+
+  /**
+   * Sets text styles for descendants of the
+   * <amp-story-grid-layer> element.
+   * @private
+   */
+  setDescendentCssTextStyles_() {
+    this.mutateElement(() => {
+      setTextBackgroundColor(this.element);
+    });
+  }
```

Comment:
Instead of doing it from these three places  (grid-layer, attachment, cta-layer), could we do just one operation from their common parent? I think moving this code to `amp-story-page.js` should work :)

Reply:
Sure np!